### DISCUSSION TITLE
Wiki JS

### DISCUSSION THREAD

**Discussion by Timur123415** [requarks/wiki#6107] [Help / Questions]
Я не понимаю почему группе "пользователь" не видно картинок на странице.
Ниже я прикрепляю скрины
![Снимок экрана 2023-01-31 104624](https://user-images.githubusercontent.com/106232712/215698667-ac1ef123-ae61-40e4-9017-84626a9d4faf.png)
![Снимок экрана 2023-01-31 104642](https://user-images.githubusercontent.com/106232712/215698695-62003a28-aae7-4f6d-8e43-d5b953060e8e.png)
![Снимок экрана 2023-01-31 104654](https://user-images.githubusercontent.com/106232712/215698699-8b5f4b29-85d3-426e-b0c1-e597acc8541c.png)
![Снимок экрана 2023-01-31 104707](https://user-images.githubusercontent.com/106232712/215698701-10ecf9b1-36d2-43b2-8bb2-605673afcd31.png)
![Снимок экрана 2023-01-31 104740](https://user-images.githubusercontent.com/106232712/215698718-fac4cc88-b5b7-4be0-a450-12773f88ff28.png)

This is how I see when I log in as a user
![Снимок экрана 2023-01-31 105059](https://user-images.githubusercontent.com/106232712/215699090-d1fefa22-a8d7-449b-94e7-17fbb5d0d7b0.png)

and here's what I see when I'm an admin
![Снимок экрана 2023-01-31 105124](https://user-images.githubusercontent.com/106232712/215699350-c75c2c9d-5745-4e7e-8cbf-4e750c5661fc.png)

I don't understand where the pictures are stored and how to make them visible to the user

**Comment by keruwell** [comment]
Скорее всего вы храните картинки в отдельной папке (например image) и вам следует добавить правило именно  к этой папке

**Comment by WikiJS-pixel** [comment]
Привет, нужно добавить правило страниц этой группе на чтение всех ассетов в wiki, потому что папки, в которых хранятся картинки не прикреплены конкретно к какой-то страничке, куда у тебя прописан этой группе доступ.

Вот так:
![image](https://github.com/user-attachments/assets/f738b800-ef59-4b8d-bd8a-371bbd8c58cb)

**Reply by Papricka95** [reply]
Спасибо большое!
